src/ui/components/RootAccounts.tsx

```diff
@@ -67,14 +67,14 @@ export function RootAccounts() {
       dispatch(addBackTab(currentTab));
       dispatch(setTab(tab));
     },
-    [currentTab]
+    [currentTab, dispatch]
   );
 
   const onBack = React.useCallback(() => {
     const tab = backTabFromConf || backTabs[backTabs.length - 1] || PAGES.ROOT;
     dispatch(removeBackTab());
     dispatch(setTab(tab));
-  }, [backTabFromConf, backTabs]);
+  }, [backTabFromConf, backTabs, dispatch]);
```

Comment:
You have a lot of fixes for the `exhaustive-deps` error, but I think we need to disable this rule. Adding additional parameters may be the cause of bugs and unnecessary deps.

Reply:
Disabling this rule doesn't make sense. Often it's hard to track all the deps used in a hook.

What do you mean by "unnecessary deps"? `dispatch` is stable in this case. And it makes no difference if you add it or not, eslint just can't know for sure. I see no problem with that.

And regarding bugs. If some effect is called more often than needed, it's not that hard to fix. Are there any other kinds of bugs you have in mind?

I think this rule is very helpful.

Reply:
I mean, the logic may not assume dependence on the added parameter, which may cause bugs. 

> eslint just can't know for sure

Therefore, this rule seems unnecessary to me. It is better to use it yourself than to write unnecessary deps for the sake of eslint.

Reply:
Discussed in private messages, I think that the rule adds complexity to hooks usage but I agree that it generally reduces the number of bugs.